dataprofiler/tests/profilers/utils.py

```diff
@@ -11,8 +12,7 @@ def set_seed(seed=None):
     :param seed:
     :return:
     """
-    np.random.seed(seed)
-    random.seed(seed)
+    settings._seed = seed
```

Comment:
this should also set the seed for np.random and random.seed in addition to the settings

Reply:
or maybe we should remove this and the set_seed func in the __init__ should set these as well.

In this case, we would likely move the set_seed into its own file and import set_seed from the file so that it could be accessed directly via `dp.set_seed`

Reply:
@granteden thoughts?

Reply:
e.g. a utils

Reply:
also utils shouldn't directly alter the `_seed` but utilize the `dp.set_seed`

Reply:
I think it should be a solution in one place, rather than top level in init and here. Because if you are just setting `_seed` here then what is the use case for the one in `__init__`?

Reply:
actually, now that i think about it, i think we leave it out of `__init__` reason being, the user can choose to specify those externally. If we alter it globally, we could be altering it for someone else.